### PR TITLE
Otp level up

### PR DESCRIPTION
**Pull request by nelsonic**
OTP levels up our skill to write distributed asynchronous applications. ⏩ 
Code for **Chapter 11** of the **Programming Phoenix Book**  📚 
organised by page number commit to help others read/debug/learn Phoenix faster. 🚀 

+ adds a service that queries the Wolfram Alpha API. 💡 

@iteles / @shouston3 / @SimonLab please take a look and merge if OK. thanks! 👍

### CODE REVIEW

**Comment by nelsonic** [comment]
This is why Erlang is sick:
image
The "***observer***" is included for ***FREE***! 😍

**Comment by nelsonic** [comment]
image

File: rumbl/priv/repo/backend_seeds.exs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+alias Rumbl.Repo
+alias Rumbl.User
```
Comment (shouston3) [inline rumbl/priv/repo/backend_seeds.exs]: I have found `alias Rumbl.{Repo, User}` to be quite a nice pattern, but I'd say not worth changing

Comment (nelsonic) [inline rumbl/priv/repo/backend_seeds.exs]: @shouston3 that _is_ a nice pattern. 👍 
but I _agree_ that keeping it "_faithful_" to the book at this point is a good plan. 😉

**Pull request merged**